aries-site/src/pages/design-tokens/overview.mdx

```diff
@@ -0,0 +1,85 @@
+import { Box, Text } from 'grommet';
+import { Example } from '../../layouts';
+import { BetaNotification } from '../../examples';
+
+<BetaNotification />
+
+## What is a design token?
+
+Design tokens codify the design decisions that make up a design system's visual style. Design tokens replace hard-coded values, such as hex codes for color or pixel values for spacing, with clear names.
+
+Tokens can represent values such as colors, typography, spacing, motion and more.
+
+An HPE Design Token consists of two main parts:
+
+1. A self-documenting name that represents the coded value: “hpe.color.brand".
+1. An associated value: #01A982.
+
+Additionally, a design token can have these optional properties:
+
+1. Type: A category for the token's value (e.g., color, dimension, shadow).
+1. Description: Concise text to describe how the token should be used.
+
+To know more about token naming, visit [How to read tokens](tokens/how-to-read-tokens).
+
+TO DO: Add visual
```

Comment:
Present as a table similar to anatomies? Might be more economical and scannable.

Reply:
Done.